client/views/voip/routes.js

```diff
@@ -0,0 +1,7 @@
+import { Meteor } from 'meteor/meteor';
+
+import { createRouteGroup } from '../../lib/createRouteGroup';
+
+export const registerVoIPRoutes = createRouteGroup('voip', '/voip', () => import('./VoIPRouter'));
+
+Meteor.startup(() => {});
```

Comment:
Should this register something? 👀

Reply:
This is a temporary page used for testing. When we have final UI, this will be replaced by correct paths and views. I dont think anything to be registered at this point of time.